### DISCUSSION TITLE
How can I modify the confirmed_at and email_confirmed_at column when using scripts to insert my data into Supabase?

### DISCUSSION THREAD

**Discussion by vietnamesekid** [supabase/supabase#22427] [Questions] [answered]
I'm facing an issue when trying to insert all accounts from my database into Supabase. However, I can't log in because some fields in the Supabase users table need to be different from null. For example, in my case, the `confirmed_at` `email_confirmed_at` field needs a value, otherwise I can't log in. Has anyone encountered a similar issue? Please help me fix it, or suggest any other solutions.

<img width="281" alt="Screenshot 2024-04-03 at 16 09 12" src="https://github.com/supabase/supabase/assets/56917374/0e72360d-5851-4bae-8590-546102b0279a">

<img width="267" alt="Screenshot 2024-04-03 at 16 11 56" src="https://github.com/supabase/supabase/assets/56917374/e1bec7a3-274b-4efd-b2bc-c54589b73e0f">

**Comment by GaryAustin1** [comment, marked as answer]
If your script is setting the user info then just set the email confirmed column to a time. 
there is also a createUser admin auth call which handles everything needed.
